### ISSUE TITLE
Feature: add the 'date and time' at the end of a failed build log

### ISSUE THREAD

**Issue by DimStar77** [openSUSE/open-build-service#884]
In case of a successful build, the termination date/time is added at the end. In case of failed build, this is not the case.

Very often though this would be an interesting indication as well on when the last attempt was (especially on projects that have rebuild=direct or rebuild=local, such as openSUSE:Factory

**Comment by DimStar77** [comment]
looking at the latest build logs, this seems to have been implemented now... 

so I'm closing this

**Comment by bgeuken** [comment]
Yup, got added here https://github.com/openSUSE/obs-build/pull/195

Glad to hear it works:)

**Comment by DimStar77** [comment]
Actually,  i have to reopen it.  There are cases where it is not added, namely when the job is killed for whatever reason (osc abortbuild, or killed because it was stuck)
